farmdata2/farmdata2_modules/fd2_example/ui/ui.html

```diff
@@ -1,6 +1,13 @@
 <!-- define a div to wrap around the app. -->
 <!-- apply v-cloak so that app is not visible until after being rendered -->
 <div id="ui" v-cloak> 
+    <!-- <div style="position: -webkit-sticky; position: sticky; top: 60px; background-color: forestgreen; padding: 3px;">Hello I am a DIV</div> -->
```

Comment:
Is this comment intended to still be here?

Reply:
Strange, I thought I removed this with my latest commit.  That comment is not intended to be there. I'll remove it in the next commit.